Samples/VirtualCamera/README.md

```diff
@@ -0,0 +1,110 @@
+# Virtual Camera project #
+### Requirement:
+&emsp;&emsp;Minimum OS version: 10.0.21364.0

```

Comment:
>10.0.21364.0 [](http://example.com/codeflow?start=32&length=12)

we may want to hold PR until the relevant SDK and build number shows up in [https://docs.microsoft.com/en-us/windows-insider/flight-hub/](https://docs.microsoft.com/en-us/windows-insider/flight-hub/)

Reply:
sure will do!